airdrop/project/contracts/distributor-contract/tests/functions/info/claim_data.rs

```diff
@@ -1,143 +1,130 @@
 mod success {
     use crate::utils::{
         interface::{
-            core::{airdrop_constructor, asset_constructor, claim},
+            core::{airdrop_constructor, asset_constructor, claim, mint_to},
             info::claim_data,
         },
+        setup::ClaimState,
```

Comment:
Personally, I would keep these imports inside the other setup import rather than separating them